src/install/aim/agent-lang/python-config.mdx

```diff
@@ -0,0 +1,129 @@
+---
+componentType: default
+headingText: Configure the Python agent
+---
+
+You installed the Python agent in the last step, but you still need to apply certain configurations so the agent can collect AI-specific data. 
```

Comment:
@hmstepanek 

For this file, I'd like to provide more context for users to help them make a decision:

- What is the max and min values for these configurations? What are the options, and why might a customer choose one over the other? For example, does higher max limits provide more granular detail, or create more noise?